lisa/stats.py

```diff
@@ -840,6 +848,35 @@ def split_row_col(group):
             collapsed_col = None
             collapse_group = {}
 
+        if backend == 'plotly':
```

Comment:
The 2nd part of this function needs to be split into 2 subfunctions, one for each backend.

Currently it's structured like that, with an arbitrary imbalance in the nesting:
```
def f():
	common()
	if cond:
		return backend1()

	return backend2()
```

Should be turned into:
```
def f():
	common()
	if cond:
		return backend1()
	else:
		return backend2()
```

with bakckend1() and backend2() split into actual separate functions or methods.